azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/collection/implementation/GroupableResourcesImpl.java

```diff
@@ -91,7 +91,7 @@ public final void deleteByResourceGroup(String groupName, String name) {
 
     @Override
     public final ServiceFuture<Void> deleteByResourceGroupAsync(String groupName, String name, ServiceCallback<Void> callback) {
-        return ServiceFuture.fromBody(deleteByResourceGroupAsync(groupName, name).<Void>toObservable(), callback);
+        return ServiceFuture.fromBody(deleteByResourceGroupAsync(groupName, name).andThen(Observable.<Void>just(null)), callback);
```

Comment:
`Completable.<Void>toObservable()` produces value-less stream, ServiceFuture.fromBody apply last() operator in this stream that cause `sequence contains no element exception`. To fix this we emit `Void` value stream as above. Instead of this we can also think of adding an overload of fromBody that takes completable (in  runtime).